src/lib/events.ts

```diff
@@ -14,13 +14,13 @@ FROM users
 WHERE ${maxFilter}
 ${ironman ? 'AND "minion.ironman" = true' : ''};`;
 
+// TODO VALIDATE/TEST THIS
```

Comment:
I didn't test it, but looks good to me. Based on the prisma queryRaw docs, should return individual arrays, and psql names the column, `count` (unlike mysql) so looks to me like it should work. Still probably worth a test